config.yml

```diff
@@ -0,0 +1,5 @@
+#Dependig on the system(Windows or Linux) change the backward or forward slash appropriately.
+Tweets: /TweetsStore/
```

Comment:
Windows can use both back and forward slashes I think. So there won't be any change required